dashboard/server/requirements.txt

```diff
@@ -3,4 +3,6 @@ flask==3.0.3
 flask-socketio
 eventlet
 paramiko
-ollama
\ No newline at end of file
+ollama
+autopep8
+pylint
```

Comment:
Do you think we could put this to a new dev requirements file? For example `requirements-dev.txt`?

Now it seems as a detail but one goal is to make the target docker images lightweight and with small installs like this it can eventually grow. So I think it is useful to start separating it now (there might be more dev tooling in the future)